web/middleware.go

```diff
@@ -47,7 +42,7 @@ func panicRecovery(next http.Handler) http.Handler {
 		defer func() {
 			if rvr := recover(); rvr != nil {
 				debug.PrintStack()
-				log.WithError(errors.New(fmt.Sprint(rvr))).Error("recovered from panic in web handling")
+				slog.Error("recovered from panic in web handling", "error", errors.New(fmt.Sprint(rvr)))
```

Comment:
I don't think `errors.New(..)` is needed here since it's just going to log as a string...